packages/manifold/contracts/edition/ManifoldERC721Edition.sol

```diff
@@ -25,11 +25,12 @@ contract ManifoldERC721Edition is CreatorExtension, ICreatorExtensionTokenURI, I
         uint256 count;
     }
 
-    mapping(address => mapping(uint256 => string)) _tokenPrefix;
-    mapping(address => mapping(uint256 => uint256)) _maxSupply;
-    mapping(address => mapping(uint256 => uint256)) _totalSupply;
-    mapping(address => mapping(uint256 => IndexRange[])) _indexRanges;
-    mapping(address => uint256) _currentSeries;
+    mapping(uint256 => string) _tokenPrefix;
```

Comment:
all these need to key by creator address and instance id otherwise we have a security hole